ci/test/pipeline.template.yml

```diff
@@ -227,19 +227,6 @@ steps:
     agents:
       queue: linux-x86_64
 
-  - id: upsert-compaction-disabled
```

Comment:
Did you mean to merge this?

Reply:
Also confused by this change, did it accidentally slip in?

Reply:
My bad, fixed!